site/en/r2/tutorials/eager/custom_training.ipynb

```diff
@@ -142,9 +142,9 @@
         "id": "wfneTXy7JcUz"
       },
       "source": [
-        "TensorFlow, however, has stateful operations built in, and these are often more pleasant to use than low-level Python representations of your state. To represent weights in a model, for example, it's often convenient and efficient to use TensorFlow variables.\n",
+        "TensorFlow, however, has stateful operations built in, and these are often more pleasant to use than low-level Python representations of your state. To represent weights in a model, for example, it's often convenient and efficient to use TensorFlow variables, which are manipulated via the `tf.Variable` class.\n",
         "\n",
-        "A Variable is an object which stores a value and, when used in a TensorFlow computation, will implicitly read from this stored value. There are operations (`tf.assign_sub`, `tf.scatter_update`, etc) which manipulate the value stored in a TensorFlow variable."
+        "A variable is an object, which stores a value and, when used in a TensorFlow computation via `tf.Variable`, will implicitly read from this stored value. There are operations (`tf.assign_sub`, `tf.scatter_update`, etc), which manipulate the value stored in a TensorFlow variable."
```

Comment:
Same here:

"A tf.Variable` is an object, which stores a value, and will implicitly read from this stored value."

Reply:
Yup! It's simpler and avoids repetition ("...variable... `variable`...") @MarkDaoust . Done